README.md

```diff
@@ -19,16 +19,16 @@ A Linux VM with 2 CPUs and 4Gb of dedicated RAM should suffice for the current l
 Written in TypeScript with React.js and built with Gatsby, combines source code with big book notes (as markdown) and other data (as json) to create a static website which needs to be uploaded to the server's file system (CI loop recommended for build validation and deployments).
 
 #### assets
-This VM runs a cronjob every 10 minutes that scans, downloads and unpacks new assets (crew images) to the local file system. There's also an nginx HTTP server that publishes the assets. Code is hosted [here](https://github.com/TemporalAgent7/asset-server).
+This VM runs a cronjob every 10 minutes that scans, downloads and unpacks new assets (crew images) to the local file system. There's also an nginx HTTP server that publishes the assets. Code is hosted [here](https://github.com/stt-datacore/asset-server).
 
 #### Image analysis
-Written in dotnet core and using OpenCV and Tesseract OCR, this is the most taxing (CPU and RAM) component of the system, used by the bot for recognizing behold (and voyage setup) screenshots. Code is hosted [here](https://github.com/TemporalAgent7/datacore-bot).
+Written in dotnet core and using OpenCV and Tesseract OCR, this is the most taxing (CPU and RAM) component of the system, used by the bot for recognizing behold (and voyage setup) screenshots. Code is hosted [here](https://github.com/stt-datacore/bot).
 
 #### site-server
-Serves the dynamic aspects of datacore.app (profile uploads / views, fleet info, crew comments). Code is hosted [here](https://github.com/TemporalAgent7/site-server).
+Serves the dynamic aspects of datacore.app (profile uploads / views, fleet info, crew comments). Code is hosted [here](https://github.com/stt-datacore/site-server).
 
 #### discord bot
-The Discord bot implementation (written in TypeScript with discord.js). See source code [here](https://github.com/TemporalAgent7/datacorebot).
+The Discord bot implementation (written in TypeScript with discord.js). See source code [here](https://github.com/stt-datacore/bot).
 
 #### DB
 A simple DB (currently PostgreSQL but configurable) that links discord user ids with uploaded profiles (where associated) and includes the crew comments.
```

Comment:
We're lazy and using SQLite now, as the weight of postgresql wasn't warranted for the amount of data we hold (precious little)

Reply:
That being the case then the assets.svg needs to be altered as well as it currently also states that we use postgresql.